docker/smqtk_wrapper_python/content_type_map.py

```diff
@@ -0,0 +1,175 @@
+__ext_to_type = {
```

Comment:
Great!  I'm sure this would be much better than what I was doing with that separate file.